documentation/sphinx/source/api-common.rst.inc

```diff
@@ -588,3 +588,48 @@
 .. |locality-get-addresses-for-key-blurb| replace::
 
     Returns a list of public network addresses as strings, one for each of the storage servers responsible for storing ``key`` and its associated value.
+
+.. |option-knob| replace::
+   
+   Sets internal tuning or debugging knobs. Available knobs could be found at https://github.com/apple/foundationdb/blob/master/fdbclient/ClientKnobs.h.
```

Comment:
My concern here was more that the format of the value you passed in has to be something specific. In particular, it is of the form "knob_name=value", which we should clarify, possibly with an example.

I'm not sure if we should link to the knobs; at the very least there are multiple files defining all of the possible knobs that can be set.

Reply:
Moved the format to the description.